.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -32,23 +32,11 @@ body:
       placeholder: Steps to Reproduce, e.g. "Go to [Matomo-URL] > Click on Visitors > Click Overview"
     validations:
       required: true
-  - type: dropdown
+  - type: input
     id: version
     attributes:
       label: Matomo Version
-      description: What version of Matomo are you running?
-      options:
-        - Matomo 3
-        - Matomo 4
-        - Matomo 5
-      default: 0
-    validations:
-      required: true
-  - type: input
-    id: matomo-minor-patch
-    attributes:
-      label: Matomo Patch or Minor Version
-      description: What patch or minor version of Matomo's release above are you running?
+      description: What version of Matomo are you running? e.g. Matomo 4 Version 1.1
```

Let's leave it without the examples. Correct one would be e.g. 4.15.1 but that has too many dots in it.

```suggestion
      description: What version of Matomo are you running?
```